### PR TITLE
Info after each question, fix various spacing & typography

### PR DESCRIPTION
**Pull request by alexgirard**

### CODE REVIEW

**Review by thgao — APPROVED** [review]
yahoo!

File: pages/index.jsx [hunk 1]
```diff
@@ -114,34 +136,53 @@ export default function Home({ initialItems }) {
...
+              hidden: true,
+              component: () => <PMQ4 item={item} />,
+              info: 'test',
             },
             {
               label: 'PM Q4',
               hidden: true,
               component: () => <PMQ4 item={item} />,
+              info: 'test',
             },
```
Comment (thgao) [inline pages/index.jsx:176]: duplicat q4

File: components/form-input/TotalSumForm.jsx [hunk 2]
```diff
@@ -28,7 +28,14 @@ export const CounterButton = ({
   disableIncrement,
 }) => (
   <Box display="flex">
-    <Box mr={1} width="100%">
+    <Box
+      mr={2}
+      width="100%"
+      maxWidth="160px"
+      textAlign="right"
```
Comment (thgao) [inline components/form-input/TotalSumForm.jsx:35]: I don't like the right aligned text I think its hard to read :(

I already fixed the button alignment so things stay in grids so I think we can go without this

Comment (alexgirard) [reply components/form-input/TotalSumForm.jsx:35]: I saw you fixed this in your pr so I'll just merge this in and merge yours after!

**Pull request merged**